Add readme with setup info

Ists-Cilveks opened the pull request:
Gives a quickstart guide for running the code on localhost

Ists-Cilveks committed:
Add readme with setup info

saizai commented on renderer/pyscript-offline/public/README.md:
````diff
@@ -0,0 +1,19 @@
+2. Open a terminal in `...\unlws\renderer\pyscript-offline\public`
+3. Start a local web server. If you have Python installed, this can be done by running
+    ```sh
+    python3 -m http.server
````
Change to "./run_local_server.sh" 😛

Ists-Cilveks replied:
I added another check at the start to try using "py" before trying "python3" or "python", because Windows sends those commands (if they're used alone) to the MS Store to install python, and so they're identified as valid commands. (if used with arguments they'll still give an error). I imagine this addition won't conflict with anything.

Now it should work for anyone that has python, and should give a proper message for non-Windows users who don't have it. Windows users without python might not always get clear directions.

saizai commented on renderer/pyscript-offline/public/README.md:
```diff
@@ -0,0 +1,19 @@
+## Setup
+
+1. Clone the repo
+2. Open a terminal in `...\unlws\renderer\pyscript-offline\public`
```
Should be forward slashes in Unix systems. Are you on Windows?

Ists-Cilveks replied:
Fixed it. Yes I'm on Windows

saizai committed:
Merge branch 'master' into readme

saizai commented on renderer/pyscript-offline/public/README.md:
````diff
@@ -0,0 +1,19 @@
+    ```
+4. Go to `http://localhost:8000` in your browser
+
+Each time the site is loaded, it will download a WebAssembly-based Python runtime (via PyScript), so expect to wait a while.
````
Not a negative for your readme, but how can prevent it going to external sites for this? I'm not entirely clear on how to cache pyscript libraries into the repo.

Ists-Cilveks committed:
Merge branch 'saizai:master' into readme

Ists-Cilveks committed:
Avoid MS Store links in run_local_server.sh

Previously running the script on Windows when `python` but not `python3` is available may fail because by default `python3` and `python` open the store. `py` should work on Windows regardless of which version is available, and should be correctly identified as unavailable on most other systems.

Ists-Cilveks committed:
Say to use run_local_server.sh in the readme

Signed-off-by: jan Talisoso <38190993+Ists-Cilveks@users.noreply.github.com>

Ists-Cilveks committed:
Use forward slashes in path in readme

Signed-off-by: jan Talisoso <38190993+Ists-Cilveks@users.noreply.github.com>

saizai commented:
Thanks!
